File: java/Product.java
public class Product extends Hardware{
    private String price;
    private Integer idProduct;

    public Product(){}

    public Product(String price, Integer idProduct){
        this.price = price;
        this.idProduct = idProduct;
    }

    public void setPrice(String price)
    {
        this.price = price;
    }

    public String getPrice()
    {
        return this.price;
    }

    public void setIdProduct(Integer idProduct)
    {
        this.idProduct = idProduct;
    }

    public Integer getIdProduct()
    {
        return this.idProduct;
    }
}

File: python/Product.py
from Hardware import Hardware

class Product(Hardware):
    price = ""
    idProduct = 0

    def __init__(self, price, idProduct):
        self.price = price
        self.idProduct = idProduct
    
    def setPrice(self, price):
        self.price = price

    def getPrice(self):
        return self.price

    def setIdProduct(self, idProduct):
        self.idProduct = idProduct

    def getIdProduct(self):
        return self.idProduct